lib/src/actions/actions.dart

```diff
@@ -2769,6 +2961,9 @@ abstract class PlateWellIDTFieldsAssign
   PlateWellIDTFieldsAssign._();
 
   static Serializer<PlateWellIDTFieldsAssign> get serializer => _$plateWellIDTFieldsAssignSerializer;
+
+  @override
+  String short_description() => "assign plate well idt fields";
```

Comment:
I tend to capitalize IDT for "user-facing" things, since the name of the company is capitalized.

Reply:
https://github.com/UC-Davis-molecular-computing/scadnano/pull/752/commits/0e2fbfca5847104ca37b9269a2466f88658d0ab0